### ISSUE TITLE
Don't allow builds with no git history

### ISSUE THREAD

**Issue by tmeasday** [chromaui/chromatic-cli#179]
If someone runs the CLI in a project with no history, show an error and fail.

In particular for GH actions where the default is to have no history.

**Comment by ndelangen** [comment]
The default in github's checkout v2 action is a history with just a single commit on it

**Comment by ndelangen** [comment]
I'm thinking what we're really after is changing this:
https://github.com/chromaui/chromatic-cli/blob/6ca39d6847c4352e21c38c927557235064e4a9f8/bin/git/git.js#L211-L214

alternatively we check if there's at least 2 commits?
```
await execGitCommand(`git log -n 1 --format="%H,%ct,%ce,%cn" --skip=1`)
```

WDYT @tmeasday ?

**Comment by tmeasday** [comment]
Something like that, although I'm not sure that's the right step to check it, I would have thought somewhere earlier maybe here-ish: https://github.com/chromaui/chromatic-cli/blob/6ca39d6847c4352e21c38c927557235064e4a9f8/bin/git/git.js#L238

**Comment by ghengeveld** [comment]
Fixed in #207
